config/init_db.sql

```diff
@@ -49,6 +49,9 @@ CREATE TABLE IF NOT EXISTS _vt.shard_metadata (
 CREATE USER 'vt_dba'@'localhost';
 GRANT ALL ON *.* TO 'vt_dba'@'localhost';
 GRANT GRANT OPTION ON *.* TO 'vt_dba'@'localhost';
+#CREATE USER 'vt_dba'@'127.0.0.1';
```

Comment:
Many people use this config in production. So, it's better to delete the commented out code.

Reply:
Sorry, oversight. Meant to remove the lines after testing. Fixed.